lib/csv_import.rb

```diff
@@ -70,6 +77,8 @@ def import_with_ldap(user_data_hash)
   def attempt_save_with_csv_data?(user_data)
     user = set_or_create_user_for_import(user_data)
 
+    user_data[:username] = user_data[:email]
```

Comment:
This should only happen if we're not using CAS authentication, otherwise we should leave the user_data alone.